packages/loader/container-loader/src/container.ts

```diff
@@ -1423,6 +1434,7 @@ export class Container
 		loadMode: IContainerLoadMode,
 		resolvedUrl: IResolvedUrl,
 		pendingLocalState?: IPendingContainerState,
+		loadToSequenceNumber?: number,
```

Comment:
Both this and `pendingLocalState` params would be better typed with an `| undefined` rather than an optional.

```suggestion
		pendingLocalState: IPendingContainerState | undefined,
		loadToSequenceNumber: number | undefined,
```

Reply:
> Does everyone (ex current users) need to understand loadToSequenceNumber option?

They should be able to ignore it unless they specifically want to use it for the functionality added in this PR